packages/web-app-files/tests/unit/components/FilesList/ResourceTiles.spec.ts

```diff
@@ -3,6 +3,12 @@ import ResourceTiles from 'web-app-files/src/components/FilesList/ResourceTiles.
 import { sortFields } from 'web-app-files/src/helpers/ui/resourceTiles'
 import { Resource } from '@ownclouders/web-client'
 import { mock } from 'jest-mock-extended'
+jest.mock('@ownclouders/web-pkg', () => ({
+  ...jest.requireActual('@ownclouders/web-pkg'),
+  useTileSize: jest.fn().mockReturnValue({
+    tileSizePixels: 10
```

Comment:
```suggestion
    tileSizePixels: 100
```

Could you please increase the tile size to 100px? Snapshot looks super ugly otherwise.